hawk/src/main/java/com/orhanobut/hawk/Utils.java

```diff
@@ -22,4 +22,11 @@ public static void checkRx() {
           "make sure that you have it in your dependencies");
     }
   }
+
+  public static void validate() {
```

Comment:
also we can change the modifier from public to package-private. In the future it will be easy to make the changes.